dotcom-rendering/scripts/deploy/build-riffraff-bundle.js

```diff
@@ -67,30 +67,41 @@ const copyDist = () => {
 	);
 };
 
+const copyScripts = () => {
+	log(' - copying scripts');
+	return cpy(['**/*'], path.resolve(target, 'rendering', 'scripts'), {
+		cwd: path.resolve(__dirname, '../../scripts'),
+		parents: true,
+		nodir: true,
+	});
+};
+
+const copyDistServer = () => {
+	log(' - copying server dist');
+	return cpy(['**/*'], path.resolve(target, 'rendering', 'dist'), {
+		cwd: path.resolve(__dirname, '../../dist'),
+		parents: true,
+		nodir: true,
+	});
+};
+
+const copyMakefile = () => {
+	log(' - copying makefile');
+	return cpy(['makefile'], path.resolve(target, 'rendering'));
+};
+
 const copyRiffRaff = () => {
 	log(' - copying riffraff yaml');
 	return cpy(['riff-raff.yaml'], target, { cwd: __dirname });
 };
 
 const zipBundle = () => {
 	log(' - zipping bundle');
-	return execa(
-		'zip',
-		['--recurse-paths', 'rendering.zip', '../', '--exclude', '.git/**\\*'],
-		{
-			shell: true,
-			maxBuffer: 200000000, // increase if you get a maxBuffer exceeded error
-		},
-	).then(() => {
-		cpy(['rendering.zip'], path.resolve(target, 'rendering', 'dist'));
+	return execa('zip', ['--move', '--recurse-paths', 'rendering.zip', '.'], {
+		cwd: path.resolve(target, 'rendering'),
 	});
 };
 
-const removeAR = () => {
```

Comment:
We no longer delete the AR folder as we're a lot more specific about what we zip into our production bundle